src/high-level.js

```diff
@@ -0,0 +1,24 @@
+import deepmerge from 'deepmerge';
+import {scaffold, questionNames as projectQuestionNames} from '@travi/project-scaffolder';
+import {questionNames as jsQuestionNames} from '@travi/javascript-scaffolder';
+
+export default function (options, javascriptScaffolderFactory) {
+  const javaScriptLanguageChoice = 'JavaScript';
+
+  return scaffold(
+    deepmerge(
+      options,
+      {
+        decisions: {[projectQuestionNames.PROJECT_TYPE]: javaScriptLanguageChoice},
+        languages: {
+          [javaScriptLanguageChoice]: javascriptScaffolderFactory({
```

Comment:
It feels awkward to pass both the option `JavaScript` and the answer `JavaScript` instead of just saying `projectScaffolder(javascriptOptions)`. Almost wonder if there's room for extending the exposed api to have one pathway that doesn't expect the user to make choices but instead just take all input and build. Definitely part of iteration, so not a problem at all, just thought I'd note since I'm the  noob and you've asked me to voice things that feel off as we morph toward some of the new ideas! Definitely like this concept, though 👍

Reply:
yeah, i can see what you mean, but at least currently, i think that is kind of necessary to keeping the balance between allowing project-scaffolder to be pretty generically usable but enabling orchestration in this type of context. part of the goal of this type of high-level scaffolder is to encapsulate that complexity and abstract it away from consumers tho. certainly open to more discussion here tho.

Reply:
somewhat related to this, i was initially thinking that i was going to need to provide a constant somewhere that provided a known string for the language key (`JavaScript` in this case) so that it would be predictable enough for this to work. so far all of those are user definable since the config is provided by the consumer (usually a cli at this point). i actually like how this ended up working out because it allows the key to still be consumer definable (the high-level scaffolder in this case) but defined as a constant in this local scope since the necessary usages are all this close to each other

Reply:
Funny you mention this last point because I had started typing a comment about "might be nice to pin the `JavaScript` string somewhere in the language scaffolder" or something, then realized it was fully encapsulated here within the local scope.